apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/charSequenceAssertions.kt

```diff
@@ -49,6 +49,15 @@ infix fun <T : CharSequence> Assert<T>.notTo(@Suppress("UNUSED_PARAMETER") conta
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
  * @throws IllegalArgumentException in case [expected] is not a [CharSequence], [Number] or [Char].
  */
+@Deprecated(
+    "Switch from Assert to Expect; will be removed with 1.0.0 -- see https://github.com/robstoll/atrium/releases/tag/v0.9.0#migration for migration hints and scripts.",
+    ReplaceWith(
```

Comment:
Please deprecate `to contain` and `notTo contain` above as well (almost everything needs to be deprecated, please ask if you are unsure about one). The equivalent call in the new infix API looks like the following `contains(o)`, `containsNot(o)`

Reply:
The reason I'm unsure about deprecating the `to contain` and `notTo contain` is because there isn't such implementation in the `cc-en_GB` files. and I don't know that the API call equivalent the `contains(o)` and `containsNot(o)`. thank you for giving me such info.

Reply:
Just ask in case you are uncertain, I am here to answer :slightly_smiling_face: (it would be too much work to write down every detail which could cause problems beforehand).
As hint, usually the structure of the files in the api-infix is the same as in api-cc-infix. This way you could have seen that `to contain` in apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/charSequenceAssertions.kt is now superseded by `contains(o)` in apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/charSequenceAssertions.kt
I hope this hint helps for further deprecations, there are other changes in api-infix (e.g. list.get is different)